IFC4x3/Properties/m/MaintenanceServiceCycle_01ShZOXG5BLO_Gv105Bz_e/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="MaintenanceServiceCycle_01ShZOXG5BLO_Gv105Bz_e" Name="MaintenanceServiceCycle" UniqueId="0172b8d8-8501-4b55-8f90-e410052fdfa8" PrimaryDataType="IfcTimeMeasure" />
```

Comment:
Seems a nicer way of how to document cycles. Consider it at `AssesmentFrequency`.